.eslintrc.json

```diff
@@ -111,7 +111,8 @@
         "user_events": false,
         "Plotly": false,
         "emoji_codes": false,
-        "drafts": false
+        "drafts": false,
+        "ui_state": false
```

Comment:
This line is already in here, up higher, so you should remove this from the commit.  We should probably just sort these things, if you want an easy cleanup task as a separate commit. :)